src/test/java/seedu/superta/logic/commands/UpdateAssignmentCommandTest.java

```diff
@@ -0,0 +1,92 @@
+package seedu.superta.logic.commands;
+
+import seedu.superta.model.assignment.Assignment;
+
+public class UpdateAssignmentCommandTest {
+
+    private static final String TUTORIAL_GROUP = "04a";
+    private static final String ASSIGNMENT_TITLE = "Lab 1";
+    private static final Double MAX_MARKS = 40.0;
+    private static Assignment assignment;
+
+    /*@Test
```

Comment:
If you are not planning to implement the tests in v1.4 I think it would be good to remove this file first.

Reply:
okies! removed